lib/react.dart

```diff
@@ -2,32 +2,33 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/**
- * A Dart library for building user interfaces.
- */
+/// A Dart library for building UI using ReactJS.
 library react;
 
+/// Top-level ReactJS [Component class](https://facebook.github.io/react/docs/top-level-api.html#react.component)
+/// which provides the [ReactJS Component API](https://facebook.github.io/react/docs/component-api.html)
 abstract class Component {
+  /// ReactJS [Component] props.
   Map props;
 
+  /// Provides access to the underlying DOM representation of the [render]ed [Component].
```

Comment:
Was this comment meant to go with the next member down, `getDOMNode`? (It has a doc comment of it's own added [in this commit](https://github.com/clairesarsam-wf/react-dart/commit/47dfd939e8705ef0fcedffba53ca4adc0177e463), so those would need to be merged)